src/components/IconPreview.svelte

```
@@ -1,36 +1,39 @@
-            <div data-testid="icon-wrapper" class="flex-grow p-5 w-full max-h-[223px] flex justify-center" style="color: {state.styles.glyphColor}; transform: scale({state.styles.iconScale}) translate({state.styles.imgX}px, {state.styles.imgY}px);">
-                {#if state.svgContent}
-                    {@html state.svgContent}
-                {:else}
-                    <img src={state.imageUrl} alt={state.styles.label} class="w-full h-full" data-testid="icon-image"/>
+    {#if $customizedIcon}
+        <div id="iconToCapture" class="relative w-[371px] h-[371px]">
+            <div class="relative w-full h-full shadow-lg rounded-[45px]" style="{backgroundStyle}" data-testid="icon-background">
+                <!-- Icon -->
+                <div data-testid="icon-wrapper" class="flex-grow p-5 w-full max-h-[223px] flex justify-center" style="color: {$customizedIcon?.styles.glyphColor}; transform: scale({$customizedIcon?.styles.iconScale}) translate({$customizedIcon?.styles.imgX}px, {$customizedIcon?.styles.imgY}px);">
+                    {#if $customizedIcon.svgContent}
+                        {@html $customizedIcon.svgContent}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)